src/cmd/user.go

```diff
@@ -98,10 +98,22 @@ var listUserCmd = &cobra.Command{
 	Short:   "Lists the users",
 	Example: `
 opslevel list user
+opslevel list user --ignore-deactivated
 opslevel list user -o json | jq 'map({"key": .Name, "value": .Role}) | from_entries'
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		resp, err := getClientGQL().ListUsers(nil)
+		var payloadVars *opslevel.PayloadVariables
```

Comment:
what about having

```suggestion
	payloadVars client.InitialPageVariablesPointer()
```

Then down in the if ignoreDeactived you can just do `WithoutDeactivedUsers`

Reply:
Comment added above `var payloadVars *opslevel.PayloadVariables`, this is subtle but intentional.
`payloadVars` starts off as `nil` which is what we need to pass to `ListUsers()` if `--ignore-deactivated` isn't set